src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/ListViewTests/UnoSamples_Tests.ListView.cs

```diff
@@ -163,5 +158,171 @@ public void ListView_ObservableCollection_Unused_Space()
 
 			Assert.AreEqual(3 * heightBefore, heightAfter);
 		}
+		
+		[Test]
+		[AutoRetry]
+		public void ListView_ExpandableItem_ExpandSingleItem()
+		{
+			Run("SamplesApp.Windows_UI_Xaml_Controls.ListView.ListView_Expandable_Item");
+
+			var checkBox = _app.Marked("CheckBox");
+			_app.WaitForElement(checkBox);
+
+			// Save initial state(not expanded)
+			UITestHelper.SetScreenshotDirectory();
```

Comment:
This should not be needed anymore, as the files are saved at the proper location now. What's the intent ?

Reply:
Removed